source/_components/switch.xiaomi_plug.markdown

```diff
@@ -0,0 +1,36 @@
+---
+layout: page
+title: "Xiaomi Smart WiFi Socket and Smart Power Strip"
+description: "Instructions how to integrate your Xiaomi Smart WiFi Socket aka Plug or Xiaomi Smart Power Strip within Home Assistant."
+date: 2017-08-26 10:18
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: xiaomi.png
+ha_category: Switch
+ha_version: 0.53
+ha_iot_class: "Local Polling"
+---
+
+The `xiaomi_plug` switch platform allows you to control the state of your Xiaomi Smart WiFi Socket aka Plug or Xiaomi Smart Power Strip.
+
+Current supported features are `on`, `off`. If the device provides the current load it will be reported.
```

Comment:
**Spelling & Grammar*:
- `Current supported features are` -> `Currently, the supported features are`
- Missing comma after `load`-> `the current load, it will be reported.`

Reply:
Thanks! :-)